File: Modul7/Soal4/PRAK604-2210817110008-Aditya Oktaviari.py
kalimat1= input() 
kalimat2= input()
nilai1= len(kalimat1) 
nilai2= len(kalimat2)
star= 0 
hastag= 0

if(nilai1 != nilai2):
    print("\nPanjang kalimat berbeda, pesan palsu")
else:
    print()
    for a in range(0, nilai1):
        if(kalimat1[a] == kalimat2[a]):
            if(kalimat1[a] == ' '):
                print(" ", end='')
            else:
                print("*", end='')  
                star += 1
        else:
            print("#", end='')  
            hastag += 1
    
    print("\n* =", star)
    print("# =", hastag)
    if star >= hastag :
        print("Pesan Asli")
    else :
        print("Pesan Palsu")

File: Modul7/Soal4/PRAK604-2210817110008-Aditya Oktaviari.c
#include <stdio.h>
#include <string.h>

int main()
{
    int  star=0, hastag=0, nilai1, nilai2, i;
    char kalimat1[50], kalimat2[50];
    scanf (" %[^\n]%*c", &kalimat1);
    scanf (" %[^\n]%*c", &kalimat2);
    nilai1= strlen(kalimat1);
    nilai2= strlen(kalimat2);

    if(nilai1!=nilai2){
        printf("\nPanjang kalimat berbeda, pesan palsu\n");
    }

    else{
        printf("\n");
        for(i=0; i< nilai1; i++){
            if(kalimat1[i]==kalimat2[i]){
                if(kalimat1[i]== ' '){
                    printf(" ");
                }
                else{
                    printf("*");
                    star++;
                }
            }
            else{
                printf("#");
                hastag++;
            }
        }
        printf ("\n* = %d", star);
        printf ("\n# = %d", hastag);

        if (star>=hastag){
            printf ("\nPesan Asli\n");
        }
        else{
            printf("\nPesan Palsu\n");
        }
    }
    return 0;
}
